package.json

```diff
@@ -14,28 +14,20 @@
     "node": ">=18.0.0"
   },
   "description": "Utilities to use in your style dictionary config",
-  "types": "dist/index.d.ts",
-  "module": "dist/index.js",
-  "main": "dist-cjs/index.js",
```

Comment:
This is legacy node stuff and since we have Node 18 as a req for this lib anyway (since sdv4 does), this doesn't make sense to have anymore.